README.rst

```diff
@@ -99,12 +99,20 @@ You can currently add the following:
 * ``:bitbucket_url:`` This will force the "Edit on Bitbucket" to the configured URL
 * ``:gitlab_url:`` This will force the "Edit on GitLab" to the configured URL
 
+Translations
+============
+
+You can help to translate the Read the Docs Sphinx Theme,
+please check out the ``TRANSLATE.rst`` file for more details.
```

Comment:
I think inside the demo docs. Are general idea is to make them include more information about the theme.